spanner/spansql/sql_test.go

```diff
@@ -230,6 +230,32 @@ func TestSQL(t *testing.T) {
 			"ALTER TABLE Ta ALTER COLUMN Ci SET OPTIONS (allow_commit_timestamp = null)",
 			reparseDDL,
 		},
+		{
+			&AlterDatabase{
+				Name: "dbname",
+				Alteration: SetDatabaseOptions{Options: DatabaseOptions{
+					VersionRetentionPeriod: func(s string) *string { return &s }("7d"),
+					OptimizerVersion:       func(i int) *int { return &i }(2),
+					EnableKeyVisualizer:    func(b bool) *bool { return &b }(true),
+				}},
+				Position: line(1),
+			},
+			"ALTER DATABASE dbname SET OPTIONS (optimizer_version=2, version_retention_period='7d', enable_key_visualizer=true, )",
```

Comment:
Do we need to remove the trailing `,`? Is this statement valid?

Reply:
You're right, Cloud Spanner does not support a trailing comma in an options list. I've updated that here and in the parsing and SQL printing.